src/shared/components/search-result/search-result.component.ts

```diff
@@ -29,9 +28,6 @@ export class SearchResultComponent implements OnChanges {
   @Input()
   caseState: CaseState;
 
-  @Input()
-  caseFilterFG: FormGroup;
```

Comment:
This has to come back.
See - RDM-1560 Fix for Work basket filters not applied for subsequent pages

Both Search and Workbasket filters are required to pass that down.